test/libsolidity/syntaxTests/bytecode_too_large.sol

```diff
@@ -0,0 +1,10 @@
......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................";
```

Comment:
Can you introduce newlines?

Reply:
Also, this is a nice test for the new code generator. It takes really long to generate the IR, not sure why.

Reply:
Also, we should add an optimizer stage that turns this into a loop.

Reply:
> Can you introduce newlines?

Is that important?

Reply:
I would say we should avoid having long lines in all files, including test files, if possible.

Reply:
I'm fine if you split it up with some vi/emacs magic, but there's no way I'd do it by hand :)